pkg/controllers/user/nodesyncer/nodessyncer.go

```diff
@@ -84,6 +86,38 @@ func (m *NodesSyncer) sync(key string, machine *v3.Node) error {
 	return nil
 }
 
+func (m *NodesSyncer) getNode(machine *v3.Node, nodes []*corev1.Node) (*corev1.Node, error) {
+	nodeName := node.GetNodeName(machine)
+	if nodeName != "" {
+		node, err := m.nodeClient.Get(nodeName, metav1.GetOptions{})
```

Comment:
Do you have to use a client, will a lister do?

Reply:
@ibuildthecloud no, will change. The old code was using the client, and I've switched it to lister, but apparently missed a couple of places